javascript-forms/form-values.js

```diff
@@ -0,0 +1,17 @@
+var contact = document.querySelector('#contact-form');
+
+contact.addEventListener('submit', formHandling);
+
+var formData = {};
```

Comment:
You should move this variable inside the function, it doesn't need to continue to exist after the form has completed being submitted.